### PR TITLE
Add close current tab button

### PR DESCRIPTION
**Pull request by codeimpl**
User request us to add close current tab button. So, this patch add the
button.

Refs: #769

### CODE REVIEW

**Comment by codeimpl** [comment]
The icon is temporary, I will update the icon soon.

File: banana_cake/java/org/banana/cake/CakeToolbarManager.java [hunk 1]
```diff
@@ -166,4 +166,11 @@ public void terminate() {
         if (activity == null) return;
         activity.finishAndRemoveTask();
     }
+
+    @Override
+    public void closeCurrentTab() {
+        ChromeTabbedActivity activity = getChromeTabbedActivity();
+        if (activity == null) return;
+        activity.onMenuOrKeyboardAction(R.id.close_tab, false);
```
Comment (romederful) [inline banana_cake/java/org/banana/cake/CakeToolbarManager.java:174]: I wonder what is the difference `activity.onMenuOrKeyboardAction(R.id.close_tab, false);` and `activity.getCurrentTabModel().closeTab(currentTab, true, false, true);`

Comment (jdragonbae) [inline banana_cake/java/org/banana/cake/CakeToolbarManager.java:174]: I wonder what is the difference `activity.onMenuOrKeyboardAction(R.id.close_tab, false);` and `activity.getCurrentTabModel().closeTab(currentTab, true, false, true);`

Comment (codeimpl) [reply banana_cake/java/org/banana/cake/CakeToolbarManager.java:174]: They are same. When we call activity.onMenuOrKeyboardAction(R.id.close_tab, false), they will call activity.getCurrentTabModel().closeTab(activity.getActivityTab(), true, false, true) internally.

**Review by romandev — COMMENTED** [review]
Please don't add merge commit

**Comment by codeimpl** [comment]
> Please don't add merge commit

sure.

**Review by romandev — APPROVED** [review]
lgtm

**Pull request merged**